thunder.install

```diff
@@ -305,3 +305,15 @@ function thunder_update_8102() {
     'access_unpublished',
   ]);
 }
+
+/**
+ * Installs the thunder_updater module.
+ */
+function thunder_update_8103() {
+
+  \Drupal::service('module_installer')->install([
+    'thunder_updater',
+  ]);
+
+  \Drupal::service('thunder_updater')->checkAllListPoints(FALSE);
```

Comment:
We need bullet-points for hooks: 8101 and 8102. Then this hook should be executed before 8101 and 8102, because bullet-points that should be checked by listed hooks should not be cleared by this hook. We can add "hook_update_dependencies" (and list that 8101 and 8102 depend on 8103). Other possible solution is to exchange hooks 8101 and 8103, so that order is correct.